stingray/events.py

```diff
@@ -511,3 +515,46 @@ def apply_deadtime(self, deadtime, inplace=False, **kwargs):
             new_ev = [new_ev, retall]
 
         return new_ev
+
+    def change_mjdref(self, new_mjdref):
+        """Change the MJD reference time (MJDREF) of the light curve.
+
+        Times will be now referred to this new MJDREF
+
+        Parameters
+        ----------
+        new_mjdref : float
+            New MJDREF
+
+        Returns
+        -------
+        new_lc : lightcurve.Lightcurve object
+            The new LC shifted by MJDREF
+        """
+        time_shift = (self.mjdref - new_mjdref) * 86400
+
+        new_ev = self.shift(time_shift)
+        new_ev.mjdref = new_mjdref
+        return new_ev
+
+    def shift(self, time_shift):
+        """
+        Shift the light curve and the GTIs in time.
```

Comment:
presumably shifts events, not light curve. :)

Reply:
Done (re-ooops)